coffee/chosen.proto.coffee

```diff
@@ -504,3 +504,14 @@ class @Chosen extends AbstractChosen
         w = f_width - 10
 
       @search_field.setStyle({'width': w + 'px'})
+
+  trigger_form_field_change:  ->
+    Chosen.trigger(@form_field, 'input')
+    Chosen.trigger(@form_field, 'change')
+
+  @trigger: (element, eventType) ->
+    if (document.createEventObject) # Old IE:
```

Comment:
`if` statements should be written without parens (I know, there are some conditions that do use parens, but they just slipped through the cracks I guess)